### PR TITLE
Q611 Add validTriangleNumber.py

### PR DESCRIPTION
**Pull request by peinanteng**

### CODE REVIEW

File: validTriangleNumber.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Solution(object):
+    def triangleNumber(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        edges = sorted(nums, reverse=True)
+        ans = 0
+        for index, longest in enumerate(edges):
```
Comment (nkukarl) [inline validTriangleNumber.py]: `longest` -> `length`?

Comment (nkukarl) [inline validTriangleNumber.py]: `index` is a reserved word in python

Comment (peinanteng) [reply validTriangleNumber.py]: sure, will change it.

Comment (peinanteng) [reply validTriangleNumber.py]: sure, will change the name to 'length'.

**Pull request merged**